### PR TITLE
(2739) Remove the legacy report CSV generator

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR
- Remove legacy report CSV generator

### Before
Screenshot 2023-01-05 at 13 36 34

### After
Screenshot 2023-01-05 at 13 36 22

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by CristinaRO** [comment]
[Remove "Legacy CSV"](https://trello.com/c/9ZNyzw3D/2739-remove-legacy-csv)

**Comment by yndajas** [comment]
Seitan - approve!

File: app/services/report/export.rb [hunk 1]
```diff
@@ -1,264 +0,0 @@
-class Report
```
Comment (Gweaton) [inline app/services/report/export.rb:1]: So happy to see this go! No more getting the wrong export report file!

File: db/schema.rb [hunk 2]
```diff
@@ -81,6 +81,7 @@
     t.integer "ispf_themes", array: true
     t.string "ispf_partner_countries", array: true
     t.uuid "linked_activity_id"
+    t.integer "has_linked_activity"
```
Comment (Gweaton) [inline db/schema.rb:84]: Has this crept in by mistake?

**Review by Gweaton — APPROVED** [review]
Thanks for this. Looks like there might be some of that annoying schema change snuck in (something to do with postgres versions maybe?), do you know how to solve that?

File: db/schema.rb (hunk 2 above)
Comment (yndajas) [reply db/schema.rb:84]: I keep getting this added to my schema. Seems like we have a ghost - any ideas how to fix it?

**Pull request merged**